pkgs/development/python-modules/eth-hash/default.nix

```diff
@@ -1,41 +1,50 @@
 {
   lib,
-  fetchFromGitHub,
   buildPythonPackage,
+  fetchFromGitHub,
+  isPyPy,
   pythonAtLeast,
   pythonOlder,
+  pycryptodome,
   pytest,
+  pytest-xdist,
   safe-pysha3,
-  pycryptodome,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "eth-hash";
-  version = "0.5.2";
-  format = "setuptools";
-  disabled = pythonOlder "3.5";
+  version = "0.7.0";
+
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
```

Comment:
Why the unnecessary whitespace change?

Reply:
Just to keep it consistent with the other files.